bin/runnerman.js

```diff
@@ -28,7 +33,12 @@ program
 
   const iterations = parseInt(program.iterations) || 1;
 
-  const suites = getFilesInFolderPerExtension(program.suite, "suite");
+  const suites = new Set();
+
+  for(const item of program.suite) {
+    var items = await getFilesInFolderPerExtension(item, "suite");
```

Comment:
```suggestion
    const items = await getFilesInFolderPerExtension(item, "suite");
```

Reply:
Resolved this point in https://github.com/FCesar/runnerman/pull/4